### ISSUE TITLE
Apple Mach-O Linker (ld) Error Group

### ISSUE THREAD

**Issue by kelvinblade** [OneSignal/react-native-onesignal#315]
Trying to set up onesignal with react native.

I followed the instructions and did npm install then add capabilities and search paths then I added the code in AppDelegate.h and AppDelegate.m.

When I tried to run in xcode, I am getting this error:

```
Apple Mach-O Linker (ld) Error Group
"_OBJC_CLASS_$_RCTOneSignal", referenced from:
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

**Comment by sunweiyang** [comment]
@kelvinblade I'm seeing the same issue, but I see you closed #315. Were you able to resolve it?

**Comment by jslok** [comment]
Same issue. Can someone please share the solution?
